.gulp.json

```diff
@@ -0,0 +1,5 @@
+{
+  "flags": {
+    "gulpfile": "gulpfile.cjs"
+  }
+}
```

Comment:
What looks for this file?  Is it a gulp thing?

Reply:
Yes, they added it a few versions back.